apps/backend/src/datasources/postgres/TechniqueDataSource.ts

```diff
@@ -199,4 +223,34 @@ export default class PostgresTechniqueDataSource
       throw new Error(`Error removing instrument(s) from technique: ${error}`);
     }
   }
+
+  async assignScientistsToTechnique(
+    scientistIds: number[],
+    techniqueId: number
+  ): Promise<boolean> {
+    const dataToInsert = scientistIds.map((scientistId) => ({
+      technique_id: techniqueId,
+      user_id: scientistId,
+    }));
+
+    return await database('technique_has_scientists')
+      .insert(dataToInsert)
```

Comment:
Please add .returning('*') to it because, without it, the result is not an array but an Result object.
```suggestion
      .insert(dataToInsert)
      .returning('*')
```